controllers/clusters/cassandra_controller.go

```diff
@@ -664,7 +678,7 @@ func (r *CassandraReconciler) handleDeleteCluster(
 	return models.ExitReconcile, nil
 }
 
-func (r *CassandraReconciler) startClusterStatusJob(c *v1beta1.Cassandra) error {
+func (r *CassandraReconciler) startSyncJob(c *v1beta1.Cassandra) error {
 	job := r.newWatchStatusJob(c)
 
 	err := r.Scheduler.ScheduleJob(c.GetJobID(scheduler.StatusChecker), scheduler.ClusterStatusInterval, job)
```

Comment:
StatusChecker -> Synchronizer

Reply:
I think we should change it after refactor of the all cluster resources

Reply:
reminder: https://github.com/instaclustr/operator/issues/661